src/Control/Monad/ST/Internal.js

```diff
@@ -58,19 +58,20 @@ exports.foreach = function (as) {
   };
 };
 
-exports.new = function (val) {
+function newSTRef(val) {
   return function () {
     return { value: val };
   };
-};
+}
+export { newSTRef as new };
```

Comment:
Interesting! Does `lebab` do this by default?

Reply:
I think this was done by @kl0tl. See https://github.com/purescript/purescript-st/pull/47/commits/5ba60073bf372c61705649ed0309212f4ccdd9b5#diff-9c608e1cbaa7f43f3029de5e7a1eb12e0bbb6a430bfe129fe3c715c82ad266c6R29